xray/utils/downloadindexer.go

```diff
@@ -30,6 +31,8 @@ const (
 	tempIndexerDirName  = "temp"
 )
 
+var indexerExtension = ""
```

Comment:
Instead of keeping only the extension as a variable here, I suggest that this variable keeps the Indexer file's name (in Windows, it will include the .exe suffix).
This way, you'll be able to avoid concatenating the extension to the file name every time you need it.